backend/api-service/src/backend_memory_test.go

```diff
@@ -116,8 +127,8 @@ func TestIMBListLocationInventoryTransactions(t *testing.T) {
 	inMemoryBackendTester.testListLocationInventoryTransactions(t)
 }
 
-func TestIMBNewItem(t *testing.T) {
-	inMemoryBackendTester.testNewItem(t)
```

Comment:
Why was this deleted?

Reply:
good catch - thank you! I don't know how that happened.